google/resource_cloudfunctions_function.go

```diff
@@ -33,8 +33,16 @@ var allowedIngressSettings = []string{
 	"ALLOW_INTERNAL_ONLY",
 }
 
+var allowedVpcConnectorEgressSettings = []string{
+	"ALL_TRAFFIC",
+	"PRIVATE_RANGES_ONLY",
+	"VPC_CONNECTOR_EGRESS_SETTINGS_UNSPECIFIED",
```

Comment:
```suggestion
```

GCP API docs are slightly misleading, the "UNSPECIFIED" option in enums will never actually appear and will generally be rejected by the API server.